packages/sns/src/governance.canister.ts

```diff
@@ -125,6 +127,22 @@ export class SnsGovernanceCanister extends Canister<SnsGovernanceService> {
     await this.manageNeuron(request);
   };
 
+  /**
+   * Start dissolving process of a neuron
+   */
+  public startDissolving = async (neuronId: NeuronId): Promise<void> => {
```

Comment:
Can you add tests for this and the wrapper?

Same for disburse if it's not there.

Reply:
Thanks for noticing. Tests were added.